crates/weaver_forge/templates/semconv_jq_fn/weaver.yaml

```diff
@@ -20,3 +20,12 @@ templates:
     filter: >
       semconv_grouped_metrics({"exclude_stability": ["experimental"]})
     application_mode: single
+  - pattern: semconv_events.json
+    filter: >
+      .groups
+      | map(select(.type == "event"))
+      | map(. + {
+          event_namespace: (if .id | index(".") then (.id | split(".") | .[0:-1] | join(".")) else "other" end)
+        })
+      | sort_by(.event_namespace, .id)
```

Comment:
Just a question: Why use `event_namespace` instead of `root_namespace`, as with the other signals? I just want to avoid the same kind of issues we had in the past with `name` and `metric_name`, which made it harder to generalize without providing significantly higher value for the UX. 

Not blocking.

Reply:
+1 to this from me.  We should avoid diverging here, although IIUC this is just testing code right?

We can clean this up later

Reply:
Yes, it is just testing code. And for events the entire "event_namespace" is important and not just the "root_namespace"...